src/skydb.js

```diff
@@ -0,0 +1,245 @@
+"use strict";
+
+const FormData = require("form-data");
+const {
+  defaultGetEntryOptions,
+  defaultSetEntryOptions,
+  defaultSkydbOptions,
+  buildSkynetJsonObject,
+  getPublicKeyfromPrivateKey,
+  max_revision,
+  raw_skylink_size,
+  decodeSkylinkBase64,
+} = require("./defaults");
+
+const { uriSkynetPrefix } = require("./utils");
+
+/**
+ *
+ * Section for Skydb v1 for node.js.
+ *
+ */
+
+/**
+ * Skydb v1
+ * Sets a JSON object at the registry entry corresponding to the publicKey and dataKey.
```

Comment:
```suggestion
 * Sets a JSON object at the registry entry corresponding to the privateKey and dataKey using SkyDB V1.
```

Reply:
I integrated it extrat so that developers and I have a better overview of which function belongs to which skydb version.

At the end, just before it's released, when we're done, I can take everything out and see how big it gets, then maybe split it into two files.

Reply:
Even if it's not big, I would split it out into two separate files for the improved organization.

Reply:
The docs here mention `publicKey` when they should say `privateKey`.